source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -587,6 +587,38 @@ def __init__(self):
             'will be discarded. Units : seconds. Must be > 0. Default value : 4h',
         )
 
+        parser_get.add_argument(
+            "bss_rent_duration",
+            type=int,
+            default=120,
+            hidden=True,
```

Comment:
TO REVIEWERS:

for the moment these params are hidden, it should be toggled to `False` as soon as it's stable